packages/navstar/buildinfo.json

```diff
@@ -4,7 +4,7 @@
     "navstar": {
       "kind": "git",
       "git": "https://github.com/dcos/navstar.git",
-      "ref": "d99eac511c39eb3070d5064fc098de374b21b834",
+      "ref": "6cb70142363c9ce2f3d59a5870560967dc46e559",
```

Comment:
It wouldn't be safer. There is only one way to avoid any problems with ref. We should ban `git push -f navstar master`.